.github/workflows/testing.yml

```
@@ -25,24 +29,36 @@
+            pwd
+            ls -la
+            echo "PYTHONPATH=$PYTHONPATH"
+
         - name: Install dependencies
           run: |
             python -m pip install --upgrade pip
-            pip install -r requirements.txt
+            pip install -r ../requirements.txt
+
+        - name: Set PYTHONPATH
+          run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)